docs/source/glossary.rst

```diff
@@ -0,0 +1,134 @@
+        A set of :term:`options<option>` that can be made for a particular :term:`version` of a
+        particular :term:`instrument`. This can be, for example, the choice of the
+        :term:`Fermi chopper` parameters.
+
+    option
+        One of the options that can be chosen for a given :term:`setting`. This is tied to a
+        particular :term:`version` of a particular :term:`instrument`. For example, if a
+        :term:`Fermi chopper` is a :term:`setting`, the "option" is one of the choices that can be
+        made for that :term:`Fermi chopper`.
+
+    model
+        A method to represent the :term:`resolution function`. There are different ways of
+        approximating the :term:`resolution function`, each taking into account different levels of
+        detail. The same model can often be used to describe multiple similar
+        :term:`instruments<instrument>` or multiple :term:`versions<version>` of an
+        :term:`instrument`, but in the context of this package, a model can only be accessed through
+        a particular :term:`version` of an :term:`instrument`.
+
+
+Neutron terminology
+^^^^^^^^^^^^^^^^^^^
+
+.. glossary::
+
+    source
+        A device that produces neutrons used for neutron experiments. In practice, this is usually
+        either a nuclear reactor or spallation source (a particle accelerator).
+
+    target
+        A material used in spallation :term:`sources<source>`, which is hit by the particle produced
+        from the accelerator. The collision emits neutrons which are used for the experiments.
+
+    moderator
+        A material used to slow down the neutrons produced by the collision with the :term:`target`.
+
+    beam
+        A collection of neutrons. In nuclear reactor :term:`sources<source>`, this is usually a
+        continuous beam (somewhat similar to a laser beam), while in spallation
+        :term:`sources<source>` this is usually a pulse (a short burst).
+
+    chopper
+        A rotating mechanical device designed to block the neutron :term:`beam` for some fraction of
+        each revolution.
+
+    Fermi chopper
+        A :term:`chopper` designed to select a particular narrow slice of energies from the wide
+        range of energies coming from the moderator. This kind of :term:`chopper` often has multiple
+        openings, allowing the user to make a choice of which energies to use or how good a
+        :term:`resolution` to obtain.
+
+    disk chopper
+        A type of :term:`chopper` shaped like a disk. Can be used for various purposes, such as
+        improving :term:`resolution` or removing contamination, etc.
+
+    sample
+        The material being studied by the neutron experiment. Unless specified otherwise, in this
+        library the term "sample" combines two things: the :term:`sample environment` and the sample
+        itself. The latter is the stuff that the user brings to be studied, in :term:`INS` usually
+        a single crystal or a powder.
```

Comment:
Liquids and heterogenous catalysis systems are also not unusual. Maybe just drop the last sentence.